File: python/dsa/data-structures/bst/b-s-t-inorder.py
class TreeNode:
    def __init__(self, value):
        self.value = value
        self.left = None
        self.right = None

def inorder(node):
    if not node: return
    inorder(node.left)
    visit(node)
    inorder(node.right)

# Main execution
root = build_tree([])
inorder(root)

File: javascript/dsa/data-structures/bst/b-s-t-inorder.js
class TreeNode {
  constructor(value) {
    this.value = value;
    this.left = null;
    this.right = null;
  }
}

function inorder(node) {
  if (!node) return;
  inorder(node.left);
  visit(node);
  inorder(node.right);
}

// Main execution
const root = buildTree([]);
inorder(root);